### DISCUSSION TITLE
Why nuclei stops before 100% and not overwrite file?

### DISCUSSION THREAD

**Discussion by mastercho** [projectdiscovery/nuclei#3588] [Q&A]
So to be sure i run same list 3 times  with --stats and -o to same file name and its always stops at 32% and never overwrite the first written file which i think its should... 

Is there any explanation for this?

![Screenshot](https://ruri.please-fuck.me/60Q_bJ0nZ.png)

**Comment by rxerium** [comment]
I think this is a known issue and there is a partial fix, see here:

https://github.com/projectdiscovery/interactsh/issues/525

**Reply by mastercho** [reply]
> I think this is a known issue and there is a partial fix, see here:
> 
> [projectdiscovery/interactsh#525](https://github.com/projectdiscovery/interactsh/issues/525)

That's on working branch where dont have interactsh issues, as you can see on screenshot its not aborted scan on first encounter

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
